mandatory/3-function-output.js

```diff
@@ -1,18 +1,21 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
-  return Math.random() * 10;
+  return Math.random() * 10; //The Math. random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1) with approximately uniform distribution over that range
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
-  return word1.concat(word2);
+  return word1.concat(word2);  //The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array.
```

Comment:
Are you sure this function is dealing with arrays?

Reply:
Thanks for mentioning it leaded me to testing it , and it works for merging arrays, but t also works for merging strings ,there will be no space between merged strings though.